### PR TITLE
Add hover background color to SSO buttons

### PR DESCRIPTION
**Pull request by aaronraimist**
Fixes https://github.com/vector-im/element-web/issues/18114

Before:

After:

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://61dc33ec48a39c8e20746fbc--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/elements/_SSOButtons.scss [hunk 1]
```diff
@@ -45,6 +45,10 @@ limitations under the License.
         }
     }
 
+   .mx_SSOButton:hover {
+        background-color: #F5F5F5;
```
Comment (t3chguy) [inline res/css/views/elements/_SSOButtons.scss:49]: this ought to really be a theme variable, this colour doesn't appear to be used anywhere else in the app so is far from great.

Comment (aaronraimist) [reply res/css/views/elements/_SSOButtons.scss]: Switched to $panel-hover which seems to be in the acceptable to use group of variables

**Review by t3chguy — APPROVED** [review]
Thanks

**Comment by t3chguy** [comment]
needs style lint fixing

**Pull request merged**